modules/DiffusionSolver/Solver/assemble_pwlc.cc

```diff
@@ -2,13 +2,14 @@
 
 #include "mesh/MeshContinuum/chi_meshcontinuum.h"
 
-//###################################################################
+// ###################################################################
 /**Assembles PWLC matrix for general cells.*/
 void chi_diffusion::Solver::CFEM_Assemble_A_and_b(chi_mesh::Cell& cell,
                                                   int group)
 {
-  auto pwl_sdm = std::static_pointer_cast<chi_math::SpatialDiscretization_PWLC>(this->discretization_);
-  const auto& fe_intgrl_values = pwl_sdm->GetUnitIntegrals(cell);
+  auto pwl_sdm = std::static_pointer_cast<
+    chi_math::spatial_discretization::PieceWiseLinearContinuous>(this->discretization_);
+  const auto& fe_intgrl_values = unit_integrals_.at(cell.global_id_);
```

Comment:
likely silly questions. please bear with me

- before we had cell. now it is cell.global_id_. I suppose that before we were passing the global ID but just called it cell or has chi_mesh::Cell changed? I do not see a change to mesh/MeshContinuum/chi_meshcontinuum.h in this PR

- before, we get the unit integrals from pwl_sdm-> . Where is unit_integrals_ in the new version